AirSim/AirLib/include/vehicles/car/api/CarApiBase.hpp

```diff
@@ -87,6 +87,19 @@ class CarApiBase : public VehicleApiBase
         }
     };
 
+    struct RefereeState {
```

Comment:
This as well as line 83 on `AirSim/AirLib/include/vehicles/car/api/CarRpcLibAdapators.hpp`:
Why have two similar structures with the same name? Is it feasablie to extract the structure to its own header files that both files include?

Reply:
This is how AirSim did it with other datastructures. I expect they did this because they want to separate the rpc encoding class (CarRpcLibAdapators) from the actual data structure used in the program (CarApiBase). We could change it but I would rather just follow the existing system design.

Reply:
:+1: